sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1134,11 +1134,12 @@ class GrpcServer(object):
 
   _DEFAULT_SHUTDOWN_TIMEOUT_SECS = 5
 
-  def __init__(self, state, provision_info):
+  def __init__(self, state, provision_info, num_workers):
     self.state = state
     self.provision_info = provision_info
+    max_workers = max(10, num_workers)
```

Comment:
How is this safe?

Reply:
no guarantee it's safe, instead, I throw out an error when we need more threads than we have.

Reply:
So should we simply remove this cap at 10?